metadata.json

```diff
@@ -22,13 +22,6 @@
         "9"
       ]
     },
-    {
-      "operatingsystem": "CentOS",
```

Comment:
did you check if there's also CentOS/Debian specific code that should be deleted?

Reply:
yes, I did check could not find any code related to above os